### DISCUSSION TITLE
Jump to a rule from the events

### DISCUSSION THREAD

**Discussion by BobSquarePants** [evilsocket/opensnitch#597] [Ideas]
Hi,

I don't know if there a dedicated place to propose suggestion... but here one.

From the Event view, I think it will be awesome if could right click on one entry and jump directly to the corresponding rule.

Cheers.

**Comment by BobSquarePants** [comment]
Never mind, I found :) -> https://github.com/evilsocket/opensnitch/issues/598

**Reply by BetweenTheKeyboardAndChair** [reply]
While possible, a simpler UI would help.  Maybe a right click menu?  (I read #598).  It is hard to find this feature since the rule column is not displayed by default in the events tab.  My most common needs are to edit or delete the rule. (The temporary rule worked, make it permanent.  Ooops, that connection is needed, delete that rule.)
